app/src/main/java/org/xwiki/android/sync/activities/SignInViewFlipper.kt

```diff
@@ -306,4 +309,9 @@ class SignInViewFlipper(
             }
         })
     }
+
+    fun View.hideKeyboard() {
```

Comment:
It is better to use `Context` as a context for this method

Reply:
Besides, it is better to extract this method into some util file

Reply:
@InsanusMokrassar In my first commit, I've used the function present in utils folder of this project, you suggested me to take a different approach